VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.html

```diff
@@ -22,9 +22,49 @@
           <app-participant-info-tooltip [participant]="participant"></app-participant-info-tooltip>
         </div>
       </div>
+        &nbsp;
+        <a id="edit-link" href="javascript:void(0);" (click)="setParticipantEdit(participant)" class="govuk-body vh-ml10">
+            <fa-icon [icon]="editIcon" *ngIf="!isEditingParticipant(participant.id)"></fa-icon>
+        </a>
     </div>
 
-    <div class="govuk-grid-column-one-half">
+    <div class="govuk-grid-column-one-third">
+        <span *ngIf="isEditingParticipant(participant.id)">
+            <div class="govuk-form-group vh-mt10">
+                <input
+                    type="text"
+                    id="participant-{{ participant.id }}-name"
+                    #participantInputName
+                    (blur)="onParticipantDisplayName(participantInputName.value)"
```

Comment:
might be worth using a form here and re-using the input validation we have for display names that we have in admin web